mitm/test/dns.test.ts

```diff
@@ -83,6 +83,18 @@ describe('DnsOverTlsSocket', () => {
       socket.close();
     }
   });
+
+  // this test has to wait for the upstream to disconnect, so skip by default
+  test.skip('should be able to lookup a record after a miss', async () => {
```

Comment:
Why have this in here if we're skipping it?

Reply:
Good question. That's how I tested that it was disconnecting, and then worked if the original socket disconnected. I guess I just didn't want to throw it away if i needed to test it again?

Reply:
Hrm... yeah, I'm not sure.